03.create/04.entities/15.purchase-button-tutorial/docs.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Tutorial: Create a Purchase Button for Marketplace Sales
+taxonomy:
+    category: docs
+---
+
+If you are selling items on the Marketplace, you can have a space in your domain where users can easily buy your items. This could be done with a display copy of your items, an image of your item, or a purchase button for users to click. 
+
+**On This Page:**
+
+- [Prerequisites](#prerequisites)
+- [Get an Item's MarketplaceID](#get-an-items-marketplace-id)
+- [Write a Purchase Item Script](#write-a-purchase-item-script)
+- [Create an Entity to Use as a Button](#create-an-entity-to-use-as-a-button)
+  
+  
+## Prerequisites
+
+Consider getting familiar with the following concepts before starting this tutorial:
+
+- [Create New Entities](../create-entities)
+- [Change How Entities Look](../entity-appearance)
+- [Define an Entity's Behavior](../entity-behavior)
+- [Get Started with Scripting](../../../script/get-started-with-scripting)
+- [Client Entity Scripts](../../../script/client-entity-scripts)
+- [Write Your Own Scripts](../../../script/write-scripts)
+- [Purchase Marketplace Items from Script](../../script/transfer-hfc-tutorial.html#purchase-marketplace-items)
+  
+## Get an Item's MarketplaceID
+If you are putting up an item (in your domain) you have on sale in the Marketplace, you need to get the item's 'marketplaceID'. This is a unique identifier that can be found by viewing the Marketplace page in a browser window.
+
+1. In your browser of choice, navigate to [www.highfidelity.com/marketplace](www.highfidelity.com/marketplace).
+2. Locate the item.
```

Comment:
Do you have to log in? Does the item have to be your item or can you use any item? We should clarify whatever the answer is in a "note". Either

NOTE: This tutorial lets you link to any item in the Marketplace, even those you don't own. If you link to another user's item, keep in mind that they will receive the proceeds from the sale.

NOTE: You can only sell items that you've created. This script will not work for items that you are not selling.

Reply:
done